### PR TITLE
Fix `at-rule-no-unknown` end positions

### PR DESCRIPTION
**Pull request by ybiquitous**
Part of #652

This change also adds a test helper function to make testing multiline strings easier.

### CODE REVIEW

File: src/rules/at-rule-no-unknown/__tests__/index.js [hunk 1]
```diff
@@ -7,195 +7,191 @@ testRule({
...
-      @funciton floo ($n) {}
-    `,
-      line: 2,
+      code: dedent`
+        @funciton floo ($n) {}
+      `,
+      line: 1,
+      column: 1,
+      endLine: 1,
+      endColumn: 10,
```
Comment (ybiquitous) [inline src/rules/at-rule-no-unknown/__tests__/index.js:112]: [note] It seems to me that one case change is enough to test end positions.

File: src/rules/at-rule-no-unknown/index.js [hunk 2]
```diff
@@ -83,8 +83,8 @@ export default function rule(primaryOption, secondaryOptions) {
             ruleName,
             result,
             node: warning.node,
-            line: warning.line,
-            column: warning.column
+            start: { line: warning.line, column: warning.column },
+            end: { line: warning.endLine, column: warning.endColumn },
```
Comment (kristerkari) [inline src/rules/at-rule-no-unknown/index.js:87]: @ybiquitous is this a backwards compatible change?

Comment (ybiquitous) [reply src/rules/at-rule-no-unknown/index.js:87]: Good point. I tried testing the minimum version of `peerDependencies.stylelint` locally:

https://github.com/stylelint-scss/stylelint-scss/blob/efd565d2e0f1bd1c70c4242a94be200795dd5e51/package.json#L40-L42

```console
$ npm i stylelint@14.5.1
...

$ npx jest at-rule-no-unknown --no-verbose
 FAIL  src/rules/at-rule-no-unknown/__tests__/index.js
  ● scss/at-rule-no-unknown › reject › [ true ] › '@funciton floo ($n) {}\n' › no description

    expect(received).toMatchObject(expected)

    - Expected  - 2
    + Received  + 0

      Object {
        "column": 1,
    -   "endColumn": 10,
    -   "endLine": 1,
        "line": 1,
        "text": "Unexpected unknown at-rule \"@funciton\" (scss/at-rule-no-unknown)",
      }

      at Object.<anonymous> (node_modules/jest-preset-stylelint/getTestRule.js:88:33)
```

It seems that the backward compatibility keeps. 👍🏼 

If possible, I think it's better to update `peerDependencies.stylelint` to `^14.7.0`, but not required.

**Comment by ybiquitous** [comment]
Don't mind. Thanks for the merge!

**Pull request merged**